src/App.tsx

```diff
@@ -43,6 +60,8 @@ const App = () => {
               iconName = focused ? 'information-circle' : 'information-circle-outline';
             } else if (route.name === 'Contributors') {
               iconName = focused ? 'list' : 'list-outline';
+            } else if (route.name === 'Password Changed Screen') {
```

Comment:
As far as a route name goes, this seems a little long/complex. Can we try some CameCasing and maybe omit the word "Screen"?

Reply:
yes will do!